### PR TITLE
qemu: Enable gtk gui when running on linux

### PR DESCRIPTION
**Pull request by afbjorklund**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

There was no GUI enabled for the Linux build, only for the Mac build.

### CODE REVIEW

File: Formula/qemu.rb [hunk 1]
```diff
@@ -61,7 +62,6 @@ def install
       --enable-vde
       --extra-cflags=-DNCURSES_WIDECHAR=1
       --disable-sdl
-      --disable-gtk
```
Comment (carlocab) [inline Formula/qemu.rb:64]: We probably want to keep passing this on macOS to avoid opportunistic linkage.

Comment (afbjorklund) [reply Formula/qemu.rb:64]: OK, can add it the bottom

**Comment by afbjorklund** [comment]
This is used when testing lima and podman machine with `brew`, to compare with the macOS version.

Since the SDL interface has been disabled (it is also disabled on Ubuntu), using GTK is the default GUI.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**